doc/admin/deployment.md

```diff
@@ -176,6 +176,13 @@ host list.
 devices, taking into account any specified network device class preference
 (ethernet or infiniband).
 
+Note that there is currently a [bug](https://github.com/pmem/ndctl/issues/130)
+in `ndctl` that prevents the NUMA affinity of an nvdimm namespace from being
+correctly reported.
+This prevents generation of dual engine configs using `dmg config generate`
+command on CentOS7 operating systems, the issue has reportedly been fixed on
+CentOS8.
```

Comment:
Hmm. This is not completely accurate. The bug is not in ndctl, it was in certain CentOS7.x kernels, and as we discussed, it is no longer a problem in CentOS7.9 (and possibly 7.8, but that's untested).

I suggest a quick doc-only update to the effect that some CentOS 7.x kernels from before the 7.9 release were known to have a defect that prevented ndctl from being able to report on the NUMA affinity for a namespace.

Given that this documentation will be referenced for the next year or so, it's important not to imply that this is an ongoing problem.